### PR TITLE
Updated MQTT bridge to run with TLS

### PR DESCRIPTION
**Pull request by emprateekshas**
Jira ID:  EP6US9TS7

- Added TLS self-signed client certificates
- Updated ssInstall and ssUpgrade scripts to run MQTT Bridge with TLS

### CODE REVIEW

File: mqtt/data/acl_bridge [hunk 1]
```diff
@@ -11,3 +11,4 @@ topic write mtconnect/#
 user hemsaw 
 topic deny mtconnect/# 
 topic readwrite control/#
+topic write monitor/#
```
Comment (skibum1869) [inline mqtt/data/acl_bridge:14]: Is there a reason the hemsaw user must write to the monitor thread? The mtconnect is a write-only standard, so the monitor topic is not ingested into the agent. So what would be the reason for the write access? Especially when it is in a denied state above?

**Review by skibum1869 — COMMENTED** [review]
The changes are good to have. I'll make sure that port 8883 is also opened on the RT1 so that we can get the information out.

File: mqtt/data/acl_bridge (hunk 1 above)
Comment (emprateekshas) [reply mqtt/data/acl_bridge:14]: There are two users `mtconnect` and `hemsaw`. `mtconnect` user is for agent to send the monitoring information and `hemsaw` is for offline programming . `hemsaw` user uses two topics, control/# and monitor/# respectively. 
  -  Through control/#, ODS devctl receives the jobs and material information sent from backend. 
  -  Through monitor/#, the ODS devctl sends the response message to the backend upon addition of jobs and materials to the database saying whether the operation was successful or  failure.

Comment (skibum1869) [reply mqtt/data/acl_bridge:14]: That makes sense; I was confusing monitor with the mtconnect topic. so we want to run them as sub-topics like `ods/control/#` or `ods/monitor/#`?

Comment (emprateekshas) [reply mqtt/data/acl_bridge:14]: We have considered `monitor/#` and `control/#` as two prefixes to differentiate the data flow.

`monitor/#` is used for the data that flows from  the machines to the backend. `monitor/#` has two subtopics, one for mtconnect and one for ods.
 - subtopic of  `monitor/#` for mtconnect would be  `monitor/<organization>/<site>/<sample/event/condition>/<device_uuid>`
 - subtopic of `monitor/#` for ods would be `monitor/<organization>/<site>/<device_uid>/devstatus`
 
`control/#` is used for  the data that flows from the backend to the ods
- subtopics for `control/#` would be `control/<organization>/<site>/<device_uid>/command`

**Pull request merged**